[2023-08-31 13:22:51,863] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2021-11-18T00:00:00+00:00 [queued]>
[2023-08-31 13:22:51,923] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2021-11-18T00:00:00+00:00 [queued]>
[2023-08-31 13:22:51,924] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:51,924] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:22:51,925] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:22:51,947] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): thrid_task> on 2021-11-18T00:00:00+00:00
[2023-08-31 13:22:51,951] {standard_task_runner.py:52} INFO - Started process 4161 to run task
[2023-08-31 13:22:51,972] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', '2021-11-18T00:00:00+00:00', '--job-id', '341', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpbt0swu4y', '--error-file', '/tmp/tmpih4a4xr8']
[2023-08-31 13:22:51,980] {standard_task_runner.py:77} INFO - Job 341: Subtask thrid_task
[2023-08-31 13:22:52,122] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task 2021-11-18T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:22:52,305] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-18T00:00:00+00:00
[2023-08-31 13:22:52,307] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:22:52,309] {bash.py:158} INFO - Running command: echo hey, I am task3 and will be running after task1 at the same time as task2!
[2023-08-31 13:22:52,378] {bash.py:169} INFO - Output:
[2023-08-31 13:22:52,461] {bash.py:173} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2023-08-31 13:22:52,482] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:22:52,537] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20211118T000000, start_date=20230831T132251, end_date=20230831T132252
[2023-08-31 13:22:52,971] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:22:53,029] {local_task_job.py:146} INFO - Task exited with return code 0
